usaspending_api/data/authoritative_agency_list.csv

```diff
@@ -75,13 +75,13 @@ CGAC AGENCY CODE,FPDS DEPARTMENT ID,AGENCY NAME,AGENCY ABBREVIATION,REGISTERED I
 011,1100,Executive Office of the President,EOP,YES,NO,OMB File,1105,Office of Administration,OA,1100,Executive Office of the President,,,,,,,,,,TRUE
 011,1100,Executive Office of the President,EOP,YES,NO,OMB File,1148,Office of Homeland Security,,1100,Executive Office of the President,,,,,,Precursor to DHS that ceased to exist after DHS was created.,,,,TRUE
 011,1100,Executive Office of the President,EOP,YES,NO,OMB File,1103,Office of Management and Budget,OMB,1100,Executive Office of the President,GPO Acronym List,,,,,,,,,TRUE
-011,1100,Executive Office of the President,EOP,YES,NO,OMB File,1110,Office of Policy Development,,1100,Executive Office of the President,,,,,,"""In 1970, President Richard M. Nixon issued an executive order that created the Office of Policy Development, a large White House office with jurisdiction over economic and domestic policy. President William J. Clinton split the office, forming the current Domestic Policy Council and the National Economic Council.""",,,,TRUE
+011,1100,Executive Office of the President,EOP,YES,NO,OMB File,1110,Office of Policy Development,,1100,Executive Office of the President,,,,,,"""In 1970	 President Richard M. Nixon issued an executive order that created the Office of Policy Development	 a large White House office with jurisdiction over economic and domestic policy. President William J. Clinton split the office	 forming the current Domestic Policy Council and the National Economic Council.""",,,,TRUE
 011,1100,Executive Office of the President,EOP,YES,NO,OMB File,1112,Office of Science and Technology Policy,OSTP,1100,Executive Office of the President,GPO Acronym List,,,,,,,,,TRUE
-011,1100,Executive Office of the President,EOP,YES,NO,OMB File,1151,"Office of the Director, IDCA",IDCA,1100,Executive Office of the President,,,,,,"""Later, in 1998, IDCA was abolished and USAID became an independent agency.""",,,,TRUE
+011,1100,Executive Office of the President,EOP,YES,NO,OMB File,1151,"Office of the Director	 IDCA",IDCA,1100,Executive Office of the President,,,,,,"""Later	 in 1998	 IDCA was abolished and USAID became an independent agency.""",,,,TRUE

```

Comment:
why are there these spaces instead of the commas that were there before?

Reply:
fixing... :-(